### ISSUE TITLE
:custom-face backquote ` expands to ' and thus expressions inside are not evaluated

### ISSUE THREAD

**Issue by lucius-martius** [conao3/leaf.el#533]
## Description
When expanding a :custom-face block containing a backquote, that backquote is replaced with a regular quote. As a consequence the color is not evaluated and custom-set-faces fails.

## Issue leaf-block
```elisp
(leaf example
  :custom-face
  (example-face . `((t(:background ,(color-expression))))))
```

## Expected output from macroexpand-1 of the leaf-block
```elisp
(prog1 'example
  (custom-set-faces
   `(example-face
     ((t
       (:background
        (\,
         (color-expression)))))
     nil "Customized with leaf in `example' block")))
```

**Comment by elfehr** [comment]
I'm successfully using the snippet below:

```
  :custom-face
  `(org-column . '((t (:height ,(face-attribute 'default :height t)))))
```

Does it help to put the backquote first and a simple quote where you have it now? I found that's the only way to get the values to expand in both `:custom` and `:custom-face`.

**Comment by lucius-martius** [comment]
@elfehr Yes, that seems to work in isolation when I try it out on an already loaded emacs instance, but fails on restart. I think what's happening is that the functions are run (and the values are inserted) at compile time or more specifically macro expansion. That however bypasses the load order in case the functions needed are not available yet and aren't set to be auto-loaded. This is different to how `use-package` does it where the code is left in place on expansion until init.el is executed.

In my specific case, what I'm trying to do is use things like `,(doom-color 'base1)` which only works when the doom-themes package is already loaded. And even if I would insert a require or autoload, the proper theme wouldn't be loaded yet.

**Comment by conao3** [comment]
It is not possible to expand the macro to be ten after the macro expansion. We need `:eval` symbol, but it is not implemented yet
https://github.com/conao3/leaf.el/issues/384
